javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,131 @@
+const firstNumber = Number(arrayOfCommand[2]); //converting into numbers for mathematical operation
+const secondNumber = Number(arrayOfCommand[4]);
+
+
+     switch (command){
+         case 'Hello my name is Rajani Busani':
+             personName = arrayOfCommand[4];
+               return `Nice to meet you ${personName}`; 
+             break;           
+         case 'What is my name?':
+             if(personName){
+               return 'your name is ' + personName+'.'; 
+             }else{
+               return'please enter your name';
+             }
+             break;
+         case 'Add fishing to my todo':                               
+             todoArray.push(todo);
+               return `${todo} added to your todo.`;
+             break;
+         case 'Add singing in the shower to my todo':
+             todoArray.push(todo);             
+               return todo+' added to your todo.';
+             break;
+         case 'Add buying the flowers to my todo':                               
+             todoArray.push(todo);
+               return `${todo} added to your todo.`;
+             break;
+         case 'Remove fishing from my todo':        
+              return removeTodoActivity(command)
+             break;
+         case 'What is on my todo?':
+              const todoString  = todoArray.join(' and ');              
+               return `You have ${todoArray.length} todos - ${todoString}.`
+             break;
+           case 'What day is it today?':
+               const today = new Date();
+               const date = today.getDate();
+               const month = today.getMonth();
+               const months = ["January","February","March","April","May","June","July",
+               "August","September","October","November","December"];
+               const year = today.getFullYear();
+                 return `${date}th of ${months[month]}, ${year}`; 
+               break;
+           case 'what is 3 + 3':
+               if (arrayOfCommand[3]=== '+'){
+                   const sum = firstNumber+secondNumber; 
+                   return sum;                  
+               }
+               break;
+            case 'what is 4 * 12':
+                if (arrayOfCommand[3]=== '*'){
+                    const multiplication = firstNumber*secondNumber;
+                    return multiplication;
+                    break;                    
+                }
+             case 'Set a timer for 4 minutes':
+                 const timeInMinutes = arrayOfCommand[4];
+                 const timeInMilliseconds = timeInMinutes * 60 * 1000;
+                // setTimeout(timer(), timeInMilliseconds) ; 
+                setTimeout(timer, timeInMilliseconds); 
+                return  `timer set for ${timeInMinutes} minutes`;
+                break; 
+
+            case 'Arithmetic operations of a and b':  //extra command to voice assistant.
+                const a = Math.floor(Math.random()*100+1);
```

Comment:
Use meaningful variable names

Reply:
i will change anna...thank you for reviewing my code.